src/libs/OptionsListUtils.js

```diff
@@ -1317,7 +1318,24 @@ function getOptions(
     const searchValue = parsedPhoneNumber.possible ? parsedPhoneNumber.number.e164 : searchInputValue.toLowerCase();
 
     // Filter out all the reports that shouldn't be displayed
-    const filteredReports = _.filter(reports, (report) => ReportUtils.shouldReportBeInOptionList(report, Navigation.getTopmostReportId(), false, betas, policies));
+    const filteredReports = _.filter(reports, (report) => {
+        const {parentReportID, parentReportActionID} = report || {};
```

Comment:
When would `report` be falsy?

Reply:
Not sure -- it happened once while I was building this but that may have been an artifact of some bad state. 

Reports here is an object, so it's possible that one of the properties of that object could be null. Especially in javascript. 

So it's mostly for safety, as it will cause the app to crash if you try and destructure `undefined` or `null`.

Reply:
I get the safety argument, but is it possible for report not to be defined, since it comes from `reports`, and if `reports` is empty the function won't run?

Reply:
If we don't know if it's possible, I don't think we should add the fallback, at least not without some logging to understand why it happens.

If we know it's possible, then let's add a comment explaining when that can happen

Reply:
Ah here is where it is null: https://github.com/Expensify/App/pull/31448#issuecomment-1888004929

Not sure why -- i guess this is called when creating the list for new chats.

Reply:
Let's add a comment explaining why it's necessary to have the fallback